docs/store.js

```diff
@@ -91,10 +91,13 @@ var firebaseCalendarStorage = function(params) {
         var calTitle = calendarObj.state.title;
         var calState = calendarObj.state;
         
-        // Store the calendar into allCalendarIds first so that the user
-        // has permission to write into calendars/calUniqueId
+        // Store the calendar into allCalendarIds, store calState
+        // TODO work on save method, how to save readers, writers, permissions, 
+        // states, etc.
         var updates = {};
         updates['users/' + userId + '/allCalendarIds/' + calUniqueId] = calTitle;
+        updates['calendars/' + calUniqueId + '/calendarState'] = calState;
```

Comment:
Not important, but you can also just define all these when you define the object literal. Not necessarily better or worse.

```
var updates = {
  'users/' + userId + '/allCalendarIds/' + calUniqueId: calTitle,
 ....
}
```